File: src/hs/logical_operators.hs
import Text.Printf

main = do
    printf "true and false is %s\n" (show (True && False))
    printf "true or false is %s\n" (show (True || False))
    printf "true xor false is %s\n" (show (True /= False))
    printf "not true  is %s" (show (not True))

File: src/c/logical_operators.c
#include <stdio.h>

int main()
{
    printf("true and false is %d\n", 1 && 0);	
	printf("true or false is %d\n", 1 || 0);
	printf("true xor false is %d\n", 1 ^ 0);
	printf("not true is %d", !1);
    
    return 0;
}